docs/roadmap.rst

```diff
@@ -136,7 +119,7 @@ Features
 -   better discovery of processes (recommended processes, featured
     processes)
 -   search function for comments and contents
--   project managers can edit permissions in all processes
+-   permissions can be added in all processes
```

Comment:
This is already possible. The point is to simplify it to a point where project managers can do it independently.